assets/src/edit-story/components/form/media.js

```diff
@@ -192,7 +192,7 @@ function MediaInput({
   };
 
   const isMenuVisible = isHovering || isFocused;
-
+  const { imgProps } = rest;
```

Comment:
Hm, do we also want to send `imgProps` to the `Container` below? Otherwise, this prop should be destructed in the component definition.

Reply:
It's definitely not needed for the `Container` but I thought it wouldn't do any harm either. I can destruct it for clarity though.